src/main/java/net/atos/zac/app/taken/converter/RESTTaakHistorieConverter.java

```diff
@@ -69,8 +71,18 @@ private RESTTaakHistorieRegel convert(final HistoricTaskLogEntry historicTaskLog
 
     private RESTTaakHistorieRegel convertData(final HistoricTaskLogEntryType type, final String data) {
         return switch (type) {
-            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(CREATED_ATTRIBUUT_LABEL);
-            case USER_TASK_COMPLETED -> new RESTTaakHistorieRegel(COMPLETED_ATTRIBUUT_LABEL);
+            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(
+                    CREATED_ATTRIBUUT_LABEL,
+                    null,
+                    CREATED_ATTRIBUUT_LABEL,
```

Comment:
How do we localize this?

Reply:
Is there a requirement to localize? You mean that the labels are now hardcoded in Dutch? That's fine for now. We only have Dutch users and there is no hard requirement to localize to EN.

Reply:
Localization is for the frontend, so I don't think it's our concern here.

Reply:
Labels are currently localized/translated. The new/old value is not. Since this is not a concern I'll resolve this conversation